src/frontend/Navigation/DefaultContainer.ts

```diff
@@ -1,4 +1,4 @@
-import { createAppContainer } from "react-navigation";
```

Comment:
New Navigation does not use `AppContainer`. Instead a Navigation Container Component is used as a wrapper to all the navigation screens/children. All the `container` files can be deleted, just leaving it in here for now for importing purposes